info.py

```diff
@@ -28,45 +30,54 @@ def print(self, *args, **kwargs):
         sys.stdout.write(chunk)
         self.chunks.extend(chunk)
 
-    def heading(self, name):
+    def heading(self, name, level):
```

Comment:
Maybe add GitHub log group marker for each top level heading

```suggestion
    def heading(self, name, level):
        if level == 0:
            self.print('::endgroup::')
            self.print('::group::pip install {}'.format(name))
```

Reply:
I'm tempted to exclude this from the file output.  The only problem I see is if people hope to capture the output the `cat` it later or somesuch and retain the groupings.  But, that doesn't seem particularly likely.

Is the `pip install something` bit actually meant to go here?  Or just part of an example from elsewhere?

Reply:
Ups... sorry. Copy/paste error. I have updated the suggestion.
Feel free to exclude it :)